src/adapters/mc/OsConfigResource.c

```diff
@@ -927,7 +931,7 @@ void MI_CALL OsConfigResource_Invoke_TestTargetResource(
     MI_UNREFERENCED_PARAMETER(methodName);
     MI_UNREFERENCED_PARAMETER(instanceName);
 
-    OsConfigResource_TestTargetResource test_result_object;
+    OsConfigResource_TestTargetResource test_result_object = {{0},{0},{0},{0},{0},{0}};
```

Comment:
```suggestion
    OsConfigResource_TestTargetResource test_result_object = {};
```

Reply:
ok, nevermind - doesn't work with older gcc's.

looks good to me.

Reply:
Do we want similar for Get and Set (lines 518 and 1127)?